doc/api/fs.md

```diff
@@ -340,6 +340,81 @@ fs.access('/etc/passwd', fs.constants.R_OK | fs.constants.W_OK, (err) => {
 });
 ```
 
+`fs.access()` should not be used to check if a file exists before using it
```

Comment:
As opposed to saying `should not be used`, perhaps something a bit softer...

```
Using `fs.acess()` to check for the existence of a file before calling `fs.open()`,
`fs.readFile()`, or `fs.writeFile()` is not recommended because doing so...
```